integration/test_redis_specific.py

```diff
@@ -45,13 +54,22 @@ def test_redis_driver_write(redis):
 
     assert data_strings == {"col1": '0'}
 
+    for key in driver.redis.scan_iter(f'*storey:{table_name}*'):
+        driver.redis.delete(key)
 
 def test_redis_driver_join(redis):
+    random_name = ''.join(random.choice(string.ascii_lowercase) for i in range(5))
```

Comment:
Avoid using random without seed in tests as it makes tests nondeterministic.

Reply:
ok